snippet_details_autocomplete.py

```diff
@@ -10,7 +10,7 @@ def run(self,edit,asset_id):
 		details = asset_wrapper.description
 		mdpopups.show_popup(
 			self.view,
-			f"{details.text if details else ''}\n```{lang}\n{content}\n```",max_width=900,
+			f"{details if details else ''}\n```{lang}\n{content}\n```",max_width=900,
```

Comment:
wouldn't this be`details.text if details else ''`

Reply:
Before .description returns annotation but now it just return str or None